tools/sync_up.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+const SWG_ROOT = path.join(__dirname, '..', 'node_modules', 'security-wg', 'vuln');
+const NPM_TARGET_ROOT = path.join(__dirname, '..', 'ecosystem');
+const CORE_TARGET_ROOT = path.join(__dirname, '..', 'core');
+
+// collect and set up ecosystem reports
+fs.readdirSync(path.join(SWG_ROOT, 'npm'))
+    .forEach((vulnname) => {
+
+        const rawContent = fs.readFileSync(path.join(SWG_ROOT, 'npm', vulnname));
+        const content = JSON.parse(rawContent);
+        const pkg = content.module_name;
+
+        const targetPath = path.join(NPM_TARGET_ROOT, pkg);
+        if(!fs.existsSync(targetPath)) {
```

Comment:
We shouldn't check before creating, right?

Reply:
I'm not sure I get this comment :/

Reply:
Sorry, I should have explained myself better there. Before creating a file or directory we should not check if it exists. We should just try to create. If it fails, it's already there. That would be more reliable, I believe.

Reply:
I am not sure, I might have to add vulnerabilities to a dir already existing